pkg/monitor/dissect.go

```diff
@@ -146,13 +152,18 @@ func getEthInfo(isOverlay bool) string {
 
 // ConnectionInfo contains tuple information and icmp code for a connection
 type ConnectionInfo struct {
-	SrcIP    net.IP
-	DstIP    net.IP
-	SrcPort  uint16
-	DstPort  uint16
-	Proto    string
-	IcmpCode string
-	Tunnel   *ConnectionInfo
+	SrcIP        net.IP
+	DstIP        net.IP
+	SrcPort      uint16
+	DstPort      uint16
+	Proto        string
+	IcmpCode     string
+	VrrpType     string
+	Vrid         string
+	Priority     string
+	IgmpType     string
+	GroupAddress string
```

Comment:
```suggestion
	VrrpType         string
	VrrpVrID         string
	VrrpPriority     string
	IgmpType         string
	IgmpGroupAddress string
```
I suggest we use consistent prefixes to clearly identify data in this struct.